web/packages/teleport/src/Users/Users.tsx

```diff
@@ -98,6 +100,40 @@ export function Users(props: State) {
           <Indicator />
         </Box>
       )}
+      {showMauInfo && (
+        <Alert
+          data-testid="users-not-mau-alert"
+          dismissible
+          onDismiss={onDismissUsersMAUNotice}
+          kind="info"
+          css={`
+            a.external-link {
+              color: ${({ theme }) => theme.colors.buttons.link.default};
+            }
+          `}
+        >
+          Note: The users displayed here are not an accurate reflection of
+          Monthly Active Users (MAU). For example, users who log in through
+          Single Sign-On (SSO) providers such as Okta will only appear here
```

Comment:
I think we should either eliminate the mention of Okta, or change this to "may" instead of "will" (or make both changes). Users may _persist_ with Okta if user sync is enabled, so we should avoid giving the impression that users _will_ disappear if the cluster has been set up for them to persist.

Also, a tiny nit: I think "Note:" is unnecessary. It's in an info box above the table; it should be clear from the presentation that the text is of note/should be noted.